Sources/NIOSSH/Keys And Signatures/NIOSSHPublicKey.swift

```diff
@@ -410,7 +410,7 @@ extension ByteBuffer {
     ///
     /// This is mostly used as part of the certified key structure.
     @discardableResult
-    mutating func writePublicKeyWithoutPrefix(_ key: NIOSSHPublicKey) -> Int {
+    mutating func writePublicKeyWithoutPrefix(_ key: NIOSSHPublicKey, prefix: String = "") -> Int {
```

Comment:
This is where I stopped making changes due to uncertainties of how to continue with determining what prefix to use.